rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -848,6 +849,29 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	SubscribeTooltip(ui->gb_DiskCacheClearing, tooltips.settings.limit_cache_size);
 	connect(ui->enableCacheClearing, &QCheckBox::stateChanged, ui->maximumCacheSize, &QSlider::setEnabled);
 
+	// Date Time Edit Box
+	m_emu_settings->EnhanceDateTimeEdit(ui->console_time_edit, emu_settings_type::ConsoleTimeOffset, tr("dd MMM yyyy HH:mm"), true, true);
+	connect(ui->console_time_reset, &QAbstractButton::clicked, [this]()
+	{
+		ui->console_time_edit->setDateTime(QDateTime::currentDateTime());
+	});
+	SubscribeTooltip(ui->gb_console_time, tooltips.settings.console_time_offset);
+	QTimer* console_time_update = new QTimer(ui->console_time_edit);
```

Comment:
I was wondering:
Why is this timer not part of the enhance method?
wouldn't you always want that timer when you use the offset mode?

And why is it set to 15000 specifically?

Reply:
The 15000 is an arbitrary time to update the display.  Given the display format is only minutes, then it could have been 60000, but then it could potentially be out of synch with system wall clock by 60 seconds.  I did originally have it 1 second, but there were situations where it would too easily exit 'edit mode' and revert to the old offset time.

Given this update time might want to be different for each offset DateTimeEdit item it just didn't feel right having it within the method, and then having to pass another argument into the method for the timeout.  Maybe now that there's a bit more code inside the lambda it does make more sense to bundle it away in the enhance method, and have it as an optional parameter.

Reply:
You could probably at least store the min and max as user data or property in the edit to avoid getting those again and also avoid possible mismatches in those values